src/db/sysdb.h

```diff
@@ -508,6 +509,15 @@ errno_t sysdb_domain_update_domain_resolution_order(
                                         const char *domain_name,
                                         const char *domain_resolution_order);
 
+errno_t
+sysdb_get_site(TALLOC_CTX *mem_ctx,
+               struct sss_domain_info *dom,
+               const char **_site);
+
+errno_t
+sysdb_set_site(struct sss_domain_info *dom,
+               const char *site);
+
```

Comment:
Do we have some tests for the other similar functions in the `sysdb_sudomains.c` module? If yes, it might be a good idea to keep the code coverage on the same level..

Reply:
Test added.